### PR TITLE
feat(tests): add tests

### PR DESCRIPTION
**Pull request by smirnovanv**
METADATA: #7

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+SKIP_CHECK=0
```
Comment (m0sk1t) [inline .env:1]: redundant

File: src/__tests__/colorText.spec.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { FgCyan, cyanText } from '../colorText';
+
+describe('should return colored text', () => {
+  let coloredText: string;
+
+  beforeEach(() => {
+    coloredText = cyanText('test colors');
```
Comment (m0sk1t) [inline src/__tests__/colorText.spec.ts:7]: const result = 'skdjfhksjdfhtest colors dkjfgnkjdfgkdfjgb';

File: src/__tests__/colorText.spec.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  });
+
+  test('should contain incoming text', () => {
+    expect(coloredText.includes('test colors')).toBe(true);
```
Comment (m0sk1t) [inline src/__tests__/colorText.spec.ts:11]: expect.equal(result, response)

File: src/__tests__/createTempFolder.spec.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import fs from 'fs';
+
+import { createTempFolder } from '../createTempFolder';
+
+describe('should create temp folder', () => {
+  test('should create temp folder', () => {
+    const folderPath = createTempFolder();
```
Comment (m0sk1t) [inline src/__tests__/createTempFolder.spec.ts:7]:
```
createTempFolder(fsLike) => {
  fsLike.mkdir('dfkjsdfjksdnfjk');
}
```

in tests 

```
const fsLike = {
  mkdir: () =>
};

createTempFolder(fsLike)
expect(fsLike.mkdir).is.called()
```

File: src/__tests__/fetchFile.spec.ts [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+import { createTempFolder } from '../createTempFolder';
+import { fetchFile } from '../fetchFile';
+
+describe('should download file', () => {
+  test('should download file', async () => {
+    const tempFolder = createTempFolder();
+    const link = 'https://filesamples.com/samples/document/txt/sample3.txt';
+
+    fetchFile(link, tempFolder).then(() => {
```
Comment (m0sk1t) [inline src/__tests__/fetchFile.spec.ts:11]: mock

**Review by m0sk1t — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (smirnovanv) [reply .env]: removed

File: src/__tests__/colorText.spec.ts (hunk 2 above)
Comment (smirnovanv) [reply src/__tests__/colorText.spec.ts]: done

File: src/__tests__/colorText.spec.ts (hunk 3 above)
Comment (smirnovanv) [reply src/__tests__/colorText.spec.ts]: done

File: src/__tests__/createTempFolder.spec.ts (hunk 4 above)
Comment (smirnovanv) [reply src/__tests__/createTempFolder.spec.ts]: done

File: src/createTempFolder.ts [hunk 6]
```diff
@@ -1,8 +1,6 @@
-import fs from 'fs';
-import os from 'os';
 import { sep } from 'path';
 
-export const createTempFolder = () => {
+export const createTempFolder = (os, fs) => {
```
Comment (m0sk1t) [inline src/createTempFolder.ts:3]: add argument types

**Review by m0sk1t — CHANGES_REQUESTED** [review]

**Review by m0sk1t — APPROVED** [review]

**Pull request merged**